INSTALL.asciidoc

```diff
@@ -45,15 +60,27 @@ cd /tmp/os-autoinst-run
 --------------------------
 7.  create a minimal "vars.json" config file
 +
+.vars.json
+[source,json]
 ---------------------------------------------------------
-cat <<EOF >vars.json
 {
-    "DISTRI":  "sle",
-    "CASEDIR": "/space/SVN/os-autoinst-distri-opensuse/",
-    "BACKEND": "s390x"
+   "ARCH" : "x86_64",   
+   "BACKEND" : "qemu",
+   "CASEDIR" : "/path/to/os-autoinst-distri-opensuse",
+   "CDMODEL" : "virtio-scsi-pci",
+   "DESKTOP" : "kde",
+   "DISTRI" : "opensuse",
+   "ISO" : "/path/to/openSUSE-Tumbleweed-DVD-x86_64-Snapshot20160715-Media.iso",
+   "PRODUCTDIR" : "/path/to/os-autoinst-distri-opensuse/products/opensuse",
+   "VNC" : 90,
 }
-EOF
 ---------------------------------------------------------
++
+You will need to correct the file paths to point to real locations. Some of
+the variables you can use are listed
+https://github.com/os-autoinst/os-autoinst/blob/master/doc/backend_vars.asciidoc[here].
+Test case specific variables are listed in the distri directories
+e.g. https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/variables[os-autoinst-distri-opensuse/variables].
```

Comment:
I am pretty sure this is already mentioned somewhere else and I don't think repeating it here is fitting